semgrep/tests/performance/targets/definitions.py

```diff
@@ -0,0 +1,5628 @@
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more information
+# Copyright (C) Nils Weiss <nils@we155.de>
+# Copyright (C) Jonas Schmidt <jonas.schmidt@st.othr.de>
+# This program is published under a GPLv2 license
+# scapy.contrib.description = Volkswagen specific definitions for UDS
+# scapy.contrib.status = skip
+from scapy.contrib.automotive.uds import UDS_RC
+from scapy.contrib.automotive.uds import UDS_RD
+from scapy.contrib.automotive.uds import UDS_RDBI
+
+
+UDS_RDBI.dataIdentifiers[0x00BD] = "Theft Protection - Download GFA-Key"
```

Comment:
Looks like this file is ~5000 assignment statements. Not sure how representative of semgrep-core parsing performance this will be.